### DISCUSSION TITLE
[SOLVED] put filters inside feed edit\add window maybe?

### DISCUSSION THREAD

**Discussion by Owyn** [martinrotter/rssguard#700] [Ideas]
I always use just one filter for one feed, and going to filters window & working there is rather cumbersome, 
I have to uncollapse all folders,
all subfolders,
find my feed,
checkbox it,
copy my usual whitelist filter there

same goes for editing filters - I have to go all the way there again and can't just edit my feed as usual for filtering purposes

Would be nice to call filters the same way we call scripts now, e.g.:

Filters field in add\edit feed window:
```
filter_preset.js params
```

and in old filters window we'd just create \ edit filter js presets which accept params

would be rather nice, what do you think?

Here's it show in my feed window redesign sketch:
![new_add_edit_feed_design](https://user-images.githubusercontent.com/1309656/164764991-56125b48-5e47-4eea-81f5-082b6a5fbaf1.png)

**Comment by martinrotter** [comment]
Do not create new discussions, I answered to previous one. I have some health problems RN + number of areas I need to pay attention to in RSS Guard is bug, so sometimes I do not respond that fast, but I for sure read all new tickets etc.

**Reply by Owyn** [reply]
Well that one was an issue, and here I wanted to discuss things (in case that wasn't the appropriate place - I've seen you convert my issue to discussion before so I got to know about this place)... but ok
